src/pages/networks/forms/NetworkAcls.tsx

```diff
@@ -69,6 +75,27 @@ const NetworkAcls: FC<Props> = ({ formik, project }) => {
             id={networlAclSelectorId}
           />
         )}
+        {formik.values.readOnly ? (
+          formik.values.security_acls.length !== 0 && (
+            <NetworkDefaultACLRead
+              values={{
+                Egress: defaultEgressIngress.Egress,
+                Ingress: defaultEgressIngress.Ingress,
+              }}
```

Comment:
```suggestion
              values={defaultEgressIngress}
```
The idea was to simplify this :)